hugo/content/3_how-tos/35_tooling_and_debugging/355_record-rosbags-for-packages.md

```diff
@@ -22,61 +22,41 @@ tags:
+| Advanced Options | The advanced option allows you to record the topics with a more granular report. Click **Show Advanced Option** to view the options.|
+| Node Name | Enter the node for which you want to record all the topics subscribed by the specific node. <validate> |
+| No. of Message | Enter the number of messages that you want to store for each topic.|
+| Max Splits | Defines the maximum number of ROS bag file splits that are saved. Older splits are deleted after the **Max Splits** count is reached. {{% notice info %}}
 Maximum 10 splits are allowed for each ROS bag job. After the message recording reaches the maximum allowed splits, the older split files are deleted to maintain the specified memory limit. 
-{{% /notice %}}
-
-  d. Split Size is the file size after which the file is split. In the **Split Size** field, type the memory size for each split of the recorded messages in MB.</br>
-
-{{% notice info %}}
-Each split size must be more than 10 MB and less than 1024 MB or 1GB. 
-{{% /notice %}}
-
-{{% notice info %}}
-
-Maximum 10 splits are allowed with the split size of more than 10 MB and less than 1 GB. However, the disk storage for the ROS bag file can not go beyond 5120 MB (5 GB).</br>
+{{% /notice %}} |
+| Split Size | Enter the memory size for each split of the recorded messages in MB.{{% notice info %}}
+Each split size must be more than 10 MB and less than 1024 MB or 1GB. </br> Maximum 10 splits are allowed with the split size of more than 10 MB and less than 1 GB. However, the disk storage for the ROS bag file can not go beyond 5120 MB (5 GB).</br>
 For example, you can configure 10 splits and each split can store up to 500 MB (10X500 MB=5 GB)  of recorded data or you configure 5 splits and each split can store up to 1024 MB (5x1024 MB=5 GB)  of recorded data. After the message recording reaches the maximum allowed splits, the older splits are deleted to maintain the specified memory limit. 
-{{% /notice %}}
-    e. Additionally, if you are creating a ROS bag job for a component that will be deployed on a device, you can specify the following parameters.
-    <ul>
-    <li>**Upload Rate**: Specify the upload rate for the ROS bag file to be uploaded in the rapyuta.io platform. </li>
-    <li>**Rate Units**: Specify the unit for the upload rate from the drop-down menu. You can select **Bytes/s**, **KB/s**, or**MB/s**.  </li>
-    <li>**Purge After**: If you want to delete the ROS bag file after it has been successfully uploaded to the rapyuta.io platform, click the toggle-button.</li>
-    </ul> 
-![device-rosbags](/images/dev-guide/rosbag-jobs/advanced-rosbags-device.png?classes=border,shadow&width=35pc)
-
-f. To add the ROS bag job and start recording the defined topics, click **Add**.
-
+{{% /notice %}}|
+| Upload Rate | (Applicable only for device runtime) Specify the upload rate for the ROS bag file to be uploaded in the rapyuta.io platform.|
```

Comment:
Does this make sense?
```suggestion
| Upload Rate (Applicable only for device runtime) | Specify the upload rate for the ROS bag file to be uploaded in the rapyuta.io platform.|
```

Reply:
It's better to have it under description

Reply:
okay